### ISSUE TITLE
Sometimes, compositing suddenly off when logging in.

### ISSUE THREAD

**Issue by tildearrow** [tildearrow/kwin-lowlatency#104]
Title says it all. OpenGLIsUnsafe set to true upon occurring.

Possible causes:
1. Unredirection unredirects the startup splash and then collapses (TODO: test).
2. KWin started in the wrong display and then it gets confused (but then compositor should be non-existing afterwards).
2. Upstream bug.

**Comment by tildearrow** [comment]
```
Jan 14 17:43:52 linux kwin_x11[1414]: Freeze in OpenGL initialization detected
Jan 14 17:43:53 linux audit[1414]: ANOM_ABEND auid=1000 uid=1000 gid=1000 ses=4 pid=1414 comm="FreezeDetector" exe="/usr/bin/kwin_x11" sig=6 res=1
Jan 14 17:44:00 linux kernel: audit: type=1701 audit(1610664233.259:123): auid=1000 uid=1000 gid=1000 ses=4 pid=1414 comm="FreezeDetector" exe="/usr/bin/kwin_x11" sig=6 res>
Jan 14 17:43:55 linux dbus-daemon[661]: [system] Activating via systemd: service name='org.freedesktop.Avahi' unit='dbus-org.freedesktop.Avahi.service' requested by ':1.52'>
Jan 14 17:43:53 linux kwin_x11[1917]: qt.qpa.xcb: QXcbConnection: XCB error: 3 (BadWindow), sequence: 180, resource id: 18874375, major code: 20 (GetProperty), minor code: 0
Jan 14 17:43:55 linux dbus-daemon[661]: [system] Activation via systemd failed for unit 'dbus-org.freedesktop.Avahi.service': Unit dbus-org.freedesktop.Avahi.service not fo>
Jan 14 17:43:53 linux kwin_x11[1917]: qt.qpa.xcb: QXcbConnection: XCB error: 3 (BadWindow), sequence: 181, resource id: 18874375, major code: 20 (GetProperty), minor code: 0
Jan 14 17:44:02 linux dbus-daemon[1273]: [session uid=1000 pid=1273] Activating via systemd: service name='org.gnome.zeitgeist.SimpleIndexer' unit='zeitgeist-fts.service' r>
Jan 14 17:43:53 linux kwin_x11[1917]: kwin_xkbcommon: XKB: Unrecognized RMLVO layout "rs" was ignored
Jan 14 17:44:02 linux dbus-daemon[1273]: [session uid=1000 pid=1273] Successfully activated service 'org.gnome.zeitgeist.Engine'
Jan 14 17:43:53 linux kwin_x11[1917]: kwin_xkbcommon: XKB: Unrecognized RMLVO layout "apl" was ignored
Jan 14 17:43:56 linux kwin_x11[1917]: kwin_core: Compositing is not possible
```

**Comment by tildearrow** [comment]
Fixed in 5.22/5.23 by increasing the timeout to 60 seconds and not writing the OpenGLIsUnsafe config value.
